### PR TITLE
Feature/add navigation

### PR DESCRIPTION
**Pull request by MarvinBlanca**
# 📲 What #

* Pull request to add navigation to fragments

# 🛠 How #

* Create button listeners on fragments  

# ✅ Acceptance criteria #

* [X] Tested locally

# ⏰ TODO #

* None

# 🔗 References #

### CODE REVIEW

**Review by danacealfonso — APPROVED** [review]

File: app/src/main/java/com/kwartracker/android/settings/ui/SettingsFragment.kt [hunk 1]
```diff
@@ -4,17 +4,45 @@ import android.os.Bundle
...
+    override fun onClick(v: View?) {
+        when (v) {
+            binding.btnBack -> {
+                findNavController().popBackStack()
+            }
+            binding.btnLoginSecurity -> {
+                findNavController().navigate(R.id.action_settingsFragment_to_loginSecurityFragment)
+            }
+            binding.btnCategories -> {
+                Toast.makeText(activity, "No Categories UI", Toast.LENGTH_SHORT).show()
```
Comment (rexendy) [inline app/src/main/java/com/kwartracker/android/settings/ui/SettingsFragment.kt]: pls add the text to strings.xml

File: app/src/main/java/com/kwartracker/android/settings/ui/categories/CategoryDetailFragment.kt [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        binding.ibEditCategory.setOnClickListener(this)
+    }
+
+    override fun onClick(v: View?) {
+        when (v) {
+            binding.btnBack -> {
+                findNavController().popBackStack()
+            }
+            binding.ibEditCategory -> {
+                Toast.makeText(activity, "Edit Category Clicked", Toast.LENGTH_SHORT).show()
```
Comment (rexendy) [inline app/src/main/java/com/kwartracker/android/settings/ui/categories/CategoryDetailFragment.kt]: pls add the text to strings.xml

File: app/src/main/java/com/kwartracker/android/settings/ui/changepassword/ChangePasswordFragment.kt [hunk 3]
```diff
@@ -4,17 +4,42 @@ import android.os.Bundle
...
+        binding.tvSave.setOnClickListener(this)
+    }
+
+    override fun onClick(v: View?) {
+        when (v) {
+            binding.btnBack -> {
+                findNavController().popBackStack()
+            }
+            binding.tvSave -> {
+                Toast.makeText(activity, "Saving changes...", Toast.LENGTH_SHORT).show()
```
Comment (rexendy) [inline app/src/main/java/com/kwartracker/android/settings/ui/changepassword/ChangePasswordFragment.kt]: pls add the text to strings.xml

File: app/src/main/res/navigation/nav_graph.xml [hunk 4]
```diff
@@ -99,12 +103,41 @@
...
+        android:name="com.kwartracker.android.settings.ui.loginsecurity.LoginSecurityFragment"
+        android:label="LoginSecurityFragment" >
+        <action
+            android:id="@+id/action_loginSecurityFragment_to_changePasswordFragment"
+            app:destination="@id/changePasswordFragment" />
+    </fragment>
+    <fragment
+        android:id="@+id/changePasswordFragment"
+        android:name="com.kwartracker.android.settings.ui.changepassword.ChangePasswordFragment"
+        android:label="ChangePasswordFragment" />
```
Comment (rexendy) [inline app/src/main/res/navigation/nav_graph.xml]: pls add labels to strings.xml

**Review by rexendy — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/kwartracker/android/profile/ui/edit/EditProfileFragment.kt [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+        when (v) {
+            binding.btnBack -> {
+                findNavController().popBackStack()
+            }
+            binding.tvChangePhoto -> {
+                Toast.makeText(activity, "Change Photo Clicked", Toast.LENGTH_SHORT).show()
+            }
+            binding.tvSave -> {
+                Toast.makeText(activity, "Saving Profile...", Toast.LENGTH_SHORT).show()
+            }
```
Comment (rexendy) [inline app/src/main/java/com/kwartracker/android/profile/ui/edit/EditProfileFragment.kt]: Pls add the text to strings.xml

File: app/src/main/java/com/kwartracker/android/settings/ui/SettingsFragment.kt (hunk 1 above)
Comment (rexendy) [inline app/src/main/java/com/kwartracker/android/settings/ui/SettingsFragment.kt]: same here, pls add to strings.xml

File: app/src/main/java/com/kwartracker/android/settings/ui/categories/CategoryDetailFragment.kt (hunk 2 above)
Comment (rexendy) [inline app/src/main/java/com/kwartracker/android/settings/ui/categories/CategoryDetailFragment.kt]: same here pls add to strings.xml

File: app/src/main/java/com/kwartracker/android/settings/ui/changepassword/ChangePasswordFragment.kt (hunk 3 above)
Comment (rexendy) [inline app/src/main/java/com/kwartracker/android/settings/ui/changepassword/ChangePasswordFragment.kt]: same here pls add to strings.xml

**Review by rexendy — CHANGES_REQUESTED** [review]

**Review by jupilot11 — APPROVED** [review]

**Review by kalvinbdr — APPROVED** [review]

**Review by rexendy — APPROVED** [review]

**Pull request merged**